models/modelling/commissioning/projects/myria/int_myria_evaluation.sql

```
@@ -0,0 +1,174 @@
+        mp.onboarded_date,
+        mp.death_date,
+        case when mp.group_value = 1 then 'Onboarded 60+ days' else 'Not onboarded' end as myria_status
+    from {{ ref('stg_myria_matched_patients') }} mp
+   --from STAGING.MYRIA.STG_MYRIA_MATCHED_PATIENTS mp
+   where file_date = (select max(file_date) from {{ ref('stg_myria_matched_patients') }})
+   --WHERE file_date = (select max(file_date) from STAGING.MYRIA.STG_MYRIA_MATCHED_PATIENTS)
+)
+ /*Build activity data (A&E): ae_encounter_summary as Counts:encounters, cost,duration
+limit to eligible group 66K rows by inner join with member spine. Added date range limits to reduce rows from 15 million!
+*/
+, ae_encounter_summary as (
```

SQLFluff (4.2.2)

[error] Found leading comma ','. Expected only trailing near line breaks.

(LT04)